LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -184,6 +188,19 @@ private String getPassword(final String credentialVaultKeyName) throws LabApiExc
         return getSecret(secretName);
     }
 
+    public boolean resetPassword(@NonNull final String upn) throws LabApiException {
+        ResetApi resetApi = new ResetApi();
+        try {
+            final CustomSuccessResponse resetResponse = resetApi.apiResetPut(upn, ResetOperation.PASSWORD.toString());
+            Thread.sleep(PASSWORD_RESET_WAIT_DURATION);
```

Comment:
I don't think we need to do the wait here, especially for failure cases. If password reset failed on the Lab Api then we should fail fast. You may want to do the wait at the end of this method and only if we know password reset succeeded on the lab